### PR TITLE
HW-5: Add JS logic

### PR DESCRIPTION
**Pull request by rogerosha**
Hello! I set the calculator to the fifth homework, it works as it must:

image

Also, if I would choose '%' or '**', the function will return error:

image

And if the result is smaller than 0 or bigger than 100, the function will return error as well:

image

image

Thanks for the review! 🌷

### CODE REVIEW

File: homeworks/ekaterina.sharova_rogerosha/5-logic/homework.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            break;
+        default:
+            result = 'Please, try again';
+    }
+    if (result > 100) {
+        result = 'Result is too big';
+    } else if (result < 0) {
+        result = 'Please, try again';
+    }
+
```
Comment (anna-vrana-moc) [inline homeworks/ekaterina.sharova_rogerosha/5-logic/homework.js:39]: Please add an error when user tries to divide by 0

**Comment by anna-vrana-moc** [comment]
Hi @rogerosha if you have extra time please:
 "2. Додайте візуальну репрезантацію результату у вигляді великих чисел під інпутами
   3. Стилі для великого текста математичниї знаків, чисел та помилок - це стилі h1 із красним кольором вашого проєкту.
   4. Помилка для завеликого результата 'Too many pokemons/games'".
   
   I am a bit late to review your work, so you have 3 days for the fixes :)

**Comment by rogerosha** [comment]
> Hi @rogerosha if you have extra time please: "2. Додайте візуальну репрезантацію результату у вигляді великих чисел під інпутами 3. Стилі для великого текста математичниї знаків, чисел та помилок - це стилі h1 із красним кольором вашого проєкту. 4. Помилка для завеликого результата 'Too many pokemons/games'".

I really tried to do this, but I can't handle it
I have several troubles:

1.  It doesn't give the input about the error below the calculator:



image


2.  I don't understand how to change lower text to 'Too many....", and also can't create the space before '=':


image


3. In general this part seems to be ok:


image


or


image

**Comment by anna-vrana-moc** [comment]
Hi @rogerosha!
I can't find big red text under the inputs( Probably you don't push the updates.
1. You can play with position relative/absolute for the input and validation error
2. It is just another text of the error. However, you can transform your text to upper/lowercase by css: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
3. I don't see big red block as on the screen
Screenshot 2023-11-07 at 03 05 21

styles should match the project's overall design (fonts, colors)
So, for now it is 1+, I canʼt raise your grade because you did not fully meet the task requirements 🥲

**Review by anna-vrana-moc — APPROVED** [review]

**Pull request merged**